IncludeToolbox/Options/FormatterOptionsPage.cs

```diff
@@ -75,13 +75,18 @@ public enum SlashMode
         #region Sorting
 
         [Category("Sorting")]
+        [DisplayName("Include delimiters in precedence regexes")]
+        [Description("If true, precedence regexes will consider delimiters (angle brackets or quotes.)")]
+        public bool RegexIncludeDelimiter { get; set; } = false;
+
+        [Category("Sorting")]
         [DisplayName("Precedence Regexes")]
         [Description("Earlier match means higher sorting priority.\n\" " + IncludeComparer.CurrentFileNameKey + "\" will be replaced with the current file name without extension.")]
         public string[] PrecedenceRegexes {
             get { return precedenceRegexes; }
             set { precedenceRegexes = value.Where(x => x.Length > 0).ToArray(); } // Remove empty lines.
         }
-        private string[] precedenceRegexes = new string[] { "$(currentFilename)\\.(?i)(h|hpp|hxx|inl|c|cpp|cxx)(?-i)$" };
+        private string[] precedenceRegexes = new string[] { "(?i)$(currentFilename)\\.(h|hpp|hxx|inl|c|cpp|cxx)(?-i)" };
```

Comment:
Moved case-insensitivity forward, and removed the end-of-line match so that the default works either way the RegexIncludeDelimiter option is set.